include/git2/version.h

```diff
@@ -7,12 +7,12 @@
 #ifndef INCLUDE_git_version_h__
 #define INCLUDE_git_version_h__
 
-#define LIBGIT2_VERSION "0.28.0"
+#define LIBGIT2_VERSION "0.29.0"
```

Comment:
There's no need to update this yet.  We update the soversion on every release that has breaking ABI changes which, realistically, is every release.  Practically speaking, this will be in conflict with our next release soon.

Reply:
I have added a new field to git_remote_callbacks which is a member of git_push_options and git_fetch_options.
From the perspective of applications compiled against the old API, member prune and all subsequent ones in git_fetch_options have an offset of 8 bytes on 64-bit systems.
I am pretty sure this breaks the ABI and would make a version bump necessary for the soname.

If you kindly would consider reserving space for this one callback in this place for your newer releases, we might not need to bump version another time if we ever get this so far that this can be merged.

Reply:
Definitely breaks the ABI but the rhythm is to update that when we release a new version.  We’ll definitely bump the soversion at that time.